addons/languages/english/pack/src/main/res/xml/terminal.xml

```diff
@@ -1,60 +1,47 @@
+
     <Row>
-        <Key android:horizontalGap="5%p" android:codes="97" android:keyLabel="a" android:keyEdgeFlags="left"/>
-        <Key android:codes="115" android:keyLabel="s"/>
-        <Key android:codes="100" android:keyLabel="d"/>
-        <Key android:codes="102" android:keyLabel="f"/>
-        <Key android:codes="103" android:keyLabel="g"/>
-        <Key android:codes="104" android:keyLabel="h"/>
-        <Key android:codes="106" android:keyLabel="j"/>
-        <Key android:codes="107" android:keyLabel="k"/>
-        <Key android:codes="108" android:keyLabel="l" android:keyEdgeFlags="right"/>
+        <Key android:codes="a" android:popupCharacters="\@àáâãāäåæą" ask:hintLabel="\@" android:keyEdgeFlags="left"/>
+        <Key android:codes="s" android:popupCharacters="$§ßśŝšṣ" ask:hintLabel="$"/>
+        <Key android:codes="d" android:popupCharacters="#%đďḏ" ask:hintLabel="# %"/>
+        <Key android:codes="f" android:popupCharacters="^\u0026" ask:hintLabel="^ \u0026"/>
+        <Key android:codes="g" android:popupCharacters="`°ĝ" ask:hintLabel="` °"/>
+        <Key android:codes="h" android:popupKeyboard="@xml/popup_querty_with_symbols_h" ask:hintLabel="_ ~"/>
+        <Key android:codes="j" android:popupCharacters="\\|ĵ" ask:hintLabel="\\ |"/>
+        <Key android:codes="k" android:popupCharacters="([{⸢" ask:hintLabel="("/>
+        <Key android:codes="l" android:popupCharacters=")]}⸣ľĺł£" ask:hintLabel=")"/>
+        <Key ask:isFunctional="true" android:codes="\u0027" android:popupCharacters="\u0022ʾʿʻ"
+             android:keyEdgeFlags="right"/>
     </Row>
-    
+
     <Row>
         <Key android:codes="-1" android:keyWidth="15%p"
-                android:isModifier="true" android:isSticky="true" android:keyEdgeFlags="left"/>
-        <Key android:codes="122" android:keyLabel="z"/>
-        <Key android:codes="120" android:keyLabel="x"/>
-        <Key android:codes="99" android:keyLabel="c"/>
-        <Key android:codes="118" android:keyLabel="v"/>
-        <Key android:codes="98" android:keyLabel="b"/>
-        <Key android:codes="110" android:keyLabel="n"/>
-        <Key android:codes="109" android:keyLabel="m"/>
+             android:isModifier="true" android:isSticky="true" android:keyEdgeFlags="left"/>
+        <Key android:codes="z" android:popupCharacters="/÷żžź" ask:hintLabel="/"/>
+        <Key android:codes="x" android:popupCharacters="*·×" ask:hintLabel="*"/>
+        <Key android:codes="c" android:popupCharacters="-—çćĉč" ask:hintLabel="−"/>
+        <Key android:codes="v" android:popupCharacters="+±" ask:hintLabel="+"/>
+        <Key android:codes="b" android:popupCharacters="\u003D"/>
+        <Key android:codes="n" android:popupCharacters="&lt;«ńňñ" ask:hintLabel="&lt;"/>
+        <Key android:codes="m" android:popupCharacters="&gt;»µ" ask:hintLabel="&gt;"/>
         <Key android:keyWidth="15%p" android:codes="-5" android:keyEdgeFlags="right" android:isRepeatable="true"/>
     </Row>
-    <!-- the terminal stuff -->
```

Comment:
why did you remove this row? I think that `ESC` and `TAB` keys are used quite a lot in terminal.

Reply:
ooops :) The terminal line is what makes the terminal layout a terminal layout! I don't know how it got removed. Definitely by mistake!

It's back now!